platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -409,11 +417,36 @@ class HangingProtocolService {
     // if loader successfully re-arranged the data with the custom strategy
     // and returned the new props, then broadcast them
     if (!loadedData) {
-      return;
+      console.warn('Not able to load data with custom strategy');
+      return false;
     }
 
     this.customImageLoadPerformed = true;
-    this._broadcastChange(this.EVENTS.CUSTOM_IMAGE_LOAD_PERFORMED, loadedData);
+    this._broadcastEvent(this.EVENTS.CUSTOM_IMAGE_LOAD_PERFORMED, loadedData);
+    return true;
+  }
+
+  _updateActiveStudyWith(list: [], propertyKey: string): void {
```

Comment:
where is `_updateActiveStudyWith` being used? I can't find anywhere being referenced ... need documentation, not sure what is happening here